docs/UserGuide.adoc

```diff
@@ -775,25 +776,28 @@ image::ui-screenshots/uimockup1_online.png[@Online Tab View, 600, 400]
 .Mark's Offline Tab View
 image::ui-screenshots/uimockup1_offline.png[@Offline Tab View, 600, 400]
 
+_{ more coming soon }_
 
-*What is the [[Dashboard]] Dashboard?*
+== Glossary
+This glossary aims to provide a definition for the special vocabulary used in this user guide.
 
-****
+[[annotation]] Annotation::
+Notes or highlights that can be made on <<offline-copy, offline copies>> of a bookmark.
+
+[[bookmark]] Bookmark::
+A saved shortcut to a website. In Mark, a bookmark can contain other information besides the website URL,
+such as reminders, <<offline-copy, offline copies>> and <<annotation, annotations>>.
+
+[[dashboard]] Dashboard::
 The default tab in view when the application starts. The dashboard
 contains a help section (command summary), favorite tags, current
 reminders, and a view of the folder hierarchy for bookmarks.
-****
-
-_{ more coming soon }_
-
-== Glossary
-This glossary aims to provide a definition for the special vocabulary used in this user guide.
 
-[[offline]] Offline::
-Refers to files stored locally on the computer.
+[[folder-hierarchy]] Folder Hierarchy::
+The structure of folders in Mark. This is displayed in the dashboard tab when the application starts.
 
-[[oneline]] Online::
-Refers to having web access, namely to browse the internet.
+[[offline-copy]] Offline copy::
+An offline copy of a bookmarked website that is stored as a HTML file on the computer.
```

Comment:
```suggestion
The copy of a bookmarked website that is used for offline viewing. It is stored as a HTML file on the computer.
```
I think can explain about the offline part, so the reader can know better about the purpose of offline-copy.